webapp/src/main/resources/properties/de.properties

```diff
@@ -142,6 +142,9 @@ search.filter.asset=Asset
 # Search filter option "plural form"
 search.filter.pluralFormOther=Der Plural von
 
+# Search filter option "tm text unit ID"
+search.filter.tmTextUnitId=Tm Text Unit ID
```

Comment:
localized files are generated, no need to edit them

Reply:
I've removed the non En files.